backend/spec/features/admin/orders/shipments_spec.rb

```diff
@@ -142,4 +142,25 @@ def ship_shipment
       end
     end
   end
+
+  context "when split shipment" do
+    let(:location_name) { "WA Warehouse" }
+    let!(:location) { create(:stock_location, name: location_name)}
+
+    it "can split item" do
+      visit spree.edit_admin_order_path(order)
+      find_all('.split-item').first.click
+      within '.stock-item-split' do
+        find('.save-split').click
+        alert_text = page.driver.browser.switch_to.alert.text
+        expect(alert_text).to include 'Please select the split destination'
+        page.driver.browser.switch_to.alert.accept
+        find('.select2-container').click
+        find(:xpath, '//body').all('.select2-drop li.select2-result', text: "#{location_name} (0 on hand)")[1].click
+        find('.save-split').click
+      end
+      sleep 5
```

Comment:
We definitely should find a way to don't need to `sleep`. The reason is not making the suite slower.